### PR TITLE
[WIP] Allow the latest version of phpdocumentor/reflection

### PR DESCRIPTION
**Pull request by BenoitLeveque**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #555
| License       | MIT
| Doc PR        | 

We can't use api-platform 1.1 with symfony 3. here is a fix to allow both version of phpdocumentator/reflection

### CODE REVIEW

**Comment by dunglas** [comment]
Do you have tested it? I can't believe that it works without having to modify the code and create an adapter like in https://github.com/phpspec/prophecy/pull/275

File: Mapping/Loader/PhpDocLoader.php [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 use Dunglas\ApiBundle\Mapping\ClassMetadata;
 use Dunglas\ApiBundle\Util\ReflectionTrait;
+use phpDocumentor\Reflection\ClassReflector;
```
Comment (dunglas) [inline Mapping/Loader/PhpDocLoader.php]: Why this change?

**Comment by BenoitLeveque** [comment]
I close the as that's doesn't fix my issue.

File: Mapping/Loader/PhpDocLoader.php (hunk 1 above)
Comment (BenoitLeveque) [inline Mapping/Loader/PhpDocLoader.php]: The use was missing
